omf/fileio/geoh5.py

```diff
@@ -1376,6 +1394,7 @@ def block_model_reordering(entity: BlockModel | VolumeElement, values: np.ndarra
     ContainerGroup: ContainerGroupConversion,
     Curve: CurveConversion,
     FloatData: ScalarDataConversion,
+    GeometricDataConstants: KnownUnsupported,
```

Comment:
mmm ok, you get this when calling ""get_conversion_map"
But then why returning an empty set?